src/Tests/Meditation.TestBootstrapManaged/EntryPoint.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Meditation.TestBootstrapManaged
+{
+    public class EntryPoint
+    {
+        public static int TestHook(string _)
+        {
+            // Report successful hook
```

Comment:
[ToFix?] What is it? Do we need an issue for this?

Reply:
Its purpose is to validate that we can successfully load an empty hook within tests. `0` means OK. No changes are necessary here.